x-pack/platform/plugins/shared/streams/server/routes/streams/significant_events/route.ts

```diff
@@ -133,16 +127,11 @@ const readSignificantEventsRoute = createServerRoute({
       throw new SecurityError(`Cannot access API on the current pricing tier`);
     }
 
-    const { streamsClient, assetClient, scopedClusterClient, licensing } = await getScopedClients({
+    const { assetClient, scopedClusterClient, licensing } = await getScopedClients({
       request,
     });
     await assertEnterpriseLicense(licensing);
 
-    const isStreamEnabled = await streamsClient.isStreamsEnabled();
```

Comment:
this fails when trying to access the /_stream/enable API without a read only role

Reply:
I've noticed we use isStreamsEnabled only in routes that do some writes